minidump/src/minidump.rs

```diff
@@ -956,16 +961,14 @@ impl Module for MinidumpModule {
     fn code_identifier(&self) -> Option<CodeId> {
         match self.codeview_info {
             Some(CodeView::Pdb70(ref raw)) => {
-                if raw.age == 0 {
+                if self.os == Os::MacOs {
                     // This is a macOS minidump. It looks like the age is
                     // typically >0 for Windows minidumps, and Breakpad sets it
                     // to 0 on macOS. We're relying on this observation in order
                     // to distinguish between these two platforms since macOS
                     // uses PDB70 instead of its own dedicated format.
                     // See the following linked comment in a Windows PDB writer:
                     // https://github.com/microsoft/microsoft-pdb/blob/082c5290e5aff028ae84e43affa8be717aa7af73/PDB/dbi/pdb.cpp#L602-L606
-                    // TODO: Figure out a way to get access to
```

Comment:
FWIW I think it should be safe to remove or rewrite this entire comment since the above check no longer relies on the age at all, and therefore the comment refers to something that no longer exists. A rewrite of the comment would note that macOS minidumps reuse PDB70 signatures instead of having their own dedicated signature, with perhaps a link to #455.